manticore/native/cpu/abstractcpu.py

```diff
@@ -589,6 +617,39 @@ def write_int(self, where, expression, size=None, force=False):
 
         self._publish('did_write_memory', where, expression, size)
 
+    def _raw_read(self, where: int, size=1) -> bytes:
+        """
+        Selects bytes from memory. Attempts to do so faster than via read_bytes.
+
+        :param where: address to read from
+        :param size: number of bytes to read
+        :return: the bytes in memory
+        """
+        map = self.memory.map_containing(where)
+        start = map._get_offset(where)
+        if type(map) is FileMap:
```

Comment:
It does make sense to use `type` in this case because we access protected members of the FileMap that aren't guaranteed to behave the same way in a subclass. Strictly speaking, the right way to do this is probably to add unprotected bulk read and write functions to filemaps and anonmaps, but for now I'll just save the type in a variable so we won't need to calculate it twice.